app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,67 @@
+TASKS = [
+  {id: 1, name: "Wash dishes", description: "Use soap and hot water"},
+  {id: 2, name: "Do laundry", description: "Wash and dry clothes"},
+  {id: 2, name: "Grocery shop", description: "Go to store and buy healthy food"}
+]
```

Comment:
When your code goes into "production", best practice is to pull code like this that you've used to debug. ;)